content/payment-providers/adyen/package.yml

```diff
@@ -0,0 +1,17 @@
+id: Vendr.Contrib.PaymentProviders.Adyen
+name: Adyen
+logo: adyen.svg
+type: PaymentProvider
+supportedBy: community
+
+packageVersion: 1.0.0
+docVersions:
+  next: 
+  current: 1.0.0
```

Comment:
Can we change all version numbers to 0.1.0 here and in the folder name of the docs version

Reply:
Actually, never mind, lets stick with 1.0.0

Reply:
Sure, feel free to update this 👍

Reply:
Yeah, I think starting from v1.0.0 would be okay, since most other payment providers also started with v1.0.0.
We can always created minor and patch versions. For Umbraco 9, I guess most payment providers will be a v2.0.0 - maybe compiled for .NET Standard.
Not sure if it will be possible to re-use same payment provider in Vendr on Umbraco 8 and 9 though?

Reply:
Yes, in Vendr v2 (which will support both v8 and v9) payment providers will be .netstandard 2.0 and so will work in both versions. You'll need to be on Vendr 2.0 though.